python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ipv6_router_ospf.yml

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ipv6_router_ospf:
+    type: dict
+    display_name: IPv6 router OSPF Configuration
+    keys:
+      process_ids:
+        type: list
+        primary_key: id
+        unique_keys:
+          - vrf
+        items:
+          type: dict
+          keys:
+            id:
+              type: int
+              convert_types:
+                - str
+              description: OSPF process ID. Must be unique across all OSPFv3 instance.
```

Comment:
The unique requirement is already mentioned in docs since this is the primary key, so you can remove that.

Reply:
Fixed!